app/config/Seeds/UsersArticlesSeed.php

```diff
@@ -0,0 +1,48 @@
+<?php
+declare(strict_types=1);
+
+use Migrations\AbstractSeed;
+
+/**
+ * UsersArticles seed.
+ */
+class UsersArticlesSeed extends AbstractSeed
+{
+    /**
+     * Run Method.
+     *
+     * Write your database seeder using this method.
+     *
+     * More information on writing seeds is available here:
+     * https://book.cakephp.org/phinx/0/en/seeding.html
+     *
+     * @return void
+     */
+    public function run(): void
+    {
+        // Insert data into the 'users' table
+        $usersData = [
+            [
+                'email' => 'cakephp@example.com',
+                'password' => 'secret',
+                'created' => date('Y-m-d H:i:s'),
+                'modified' => date('Y-m-d H:i:s'),
+            ],
+        ];
+        $this->table('users')->insert($usersData)->save();
+
+        // Insert data into the 'articles' table
+        $articlesData = [
+            [
+                'user_id' => 1,
+                'title' => 'First Post',
+                'slug' => 'first-post',
+                'body' => 'This is the first post.',
+                'published' => 1,
+                'created' => date('Y-m-d H:i:s'),
+                'modified' => date('Y-m-d H:i:s'),
+            ],
+        ];
+        $this->table('articles')->insert($articlesData)->save();
+    }
```

Comment:
Please refer https://book.cakephp.org/migrations/3/en/index.html#seed-seeding-your-database

> Be aware that, as opposed to migrations, seeders are not tracked, which means that the same seeder can be applied multiple times.

How do we handle/prevent the error if we execute the command `docker compose run --rm php-app bin/cake migrations seed` multiple times?

Reply:
Usually the seeders are runned manually by some user carefuly and also it can be one by one,  Im guessing that they prevent automatically due to keys and contrstraints conditions,
Added a try catch block if so.